manticore/native/models.py

```diff
@@ -113,14 +117,14 @@ def strcmp(state, s1, s2):
     return ret
 
 
-def strlen(state, s):
+def strlen(state: State, s: Union[int, BitVec]):
     """
     strlen symbolic model.
 
     Algorithm: Walks from end of string not including NULL building ITE tree when current byte is symbolic.
 
     :param State state: current program state
-    :param int s: Address of string
+    :param s: Address of string
     :return: Symbolic strlen result
     :rtype: Expression or int
```

Comment:
I removed the int and added the return type hint to both strcpy and strcmp. But I used Union[int, BitVec] instead of Union[int, Expression] because the linter complained about Expression when I tried to use that before